recipes/foonathan-memory/all/conanfile.py

```diff
@@ -35,7 +35,8 @@ class FoonathanMemoryConan(ConanFile):
 
     def export_sources(self):
         for p in self.conan_data.get("patches", {}).get(self.version, []):
-            copy(self, p["patch_file"], self.recipe_folder, self.export_sources_folder)
+            copy(self, p["patch_file"], self.recipe_folder,
+                 self.export_sources_folder)
```

Comment:
There's a helper in 1.52 called `export_conandata_patches` that we could use here :)